desktop/apps/artwork/components/partner_stub/index.coffee

```diff
@@ -7,6 +7,7 @@ module.exports = ($el) ->
       e.preventDefault()
       @modal = new InquireViaPhoneModalView
         width: '500px'
+        context: 'show phone number'
```

Comment:
@anipetrov could you confirm this is the context name we'd like to be passed? Also, it seems that we use `acquisition_initiative` as well. Does  that also need to be specified like we do for http://www.artsy.net/collect-art?